ganttproject/src/biz/ganttproject/impex/csv/CsvWriter.java

```diff
@@ -0,0 +1,8 @@
+package biz.ganttproject.impex.csv;
+
+/**
+ * @author akurutin on 04.04.2017.
+ */
+public interface CsvWriter extends CommonWriter {
```

Comment:
I believe you can just remove this interface and its counterpart `XlsWriter` -- they do not add anything new to their common interface